groupdocs-total/GroupDocs.Total-Angular
# Improving presentation of the document footer and header.

**Problem:**
We facing with not correct presentation of the document footer, the problem was described in https://github.com/groupdocs-editor/GroupDocs.Editor-for-.NET-WebForms/issues/8#issuecomment-567383027.

**Solution:**
Using additional styles for inner document containers and concrete height for document wrapper we able to stick footer to the document bottom.

_Note:_
It seems like partial solution, because we can't ship universal fix for arbitrary footer content.

**Tests:**
1. Check how `*.doc/*.xls`-files looks like in the Editor app.
2. Check how `*.doc/*.xls/*.ppt/*.pdf/*img`-files looks like in the Viewer app.

**Screenshots:**
_With 67% browser zoom:_
![image](https://user-images.githubusercontent.com/17431807/74654932-751ffc80-519c-11ea-9573-33a1cec580e8.png)
